hypervisor/arch/x86/guest/vlapic.c

```diff
@@ -2575,3 +2575,14 @@ void vlapic_set_apicv_ops(void)
 		apicv_ops = &apicv_basic_ops;
 	}
 }
+
+/**
+ * @pre vm != NULL 
+ * @pre vm->vmid < CONFIG_MAX_VM_NUM
+ */
+bool is_lapic_pt_enabled(struct acrn_vm *vm)
```

Comment:
move is_lapic_pt_enabled() to vm.c?

Reply:
since this API depends on is_x2apic_enabled and tied to mode of vLAPIC, I placed it here in vlapic.c

Reply:
for modularization, we may want to remove vm reference in vlapic.c.